std_msgs/Header header
exploration/MarkerState[] states
